samples/local_attestation/common/attestation.cpp

```diff
@@ -134,32 +139,24 @@ bool Attestation::attest_local_report(
 
     // Check the enclave's product id and security version
     // See enc.conf for values specified when signing the enclave.
-    if (parsed_report.identity.product_id[0] != 1)
+    if (claims[5].value[0] != 1)
```

Comment:
ditto.

Reply:
+1.  Iterate through the claims and switch on the claim name.